### ISSUE TITLE
[SI-ERROR-???] 1.3.1 Info and Relationships - The heading is missing text

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#931]
This was reported on http://www.colorado.edu/ecenter/, but I think it is an issue for any slider block.

The javascript ends up adding code to the page that looks like...

`<h3 style="z-index: 2147483646; position: relative;">&nbsp;</h3>`

Siteimprove sees that as an empty h3.  This seems easy enough to fix by using a div tag instead of heading tag and only switching it to a heading when it's populated, but I'm not that familiar with this js so I might be missing why that wouldn't be possible and/or would just create other accessibility issues.
